### ISSUE TITLE
issues selecting particular sciunit artifact on repeat

### ISSUE THREAD

**Issue by nicole-brewer** [depaul-dice/Flinc#6]
## tldr
- [ ] **bug fix**: recompile a more up to date version of sciunit to be included in Flinc
- [ ] **feature request**: the ability to select certain sciunit artifacts when running the repeat kernel

## Use Case 
I'm using FLINC to run and repeat several different notebooks. Audit and repeat work just fine when I execute the first notebook. I shut down the repeat kernel and move on to a totally different second notebook. The audit kernel works just fine once again, but the repeat kernel fails. @sranasir suggested this is because the repeat kernel is defaulting to using the `e1` artifact instead of using the `e2` artifact.

### Feature Request

The ***ideal scenario*** would to be able to select the sciunit artifact of my choosing. So I could select `e2` when selecting a kernel. For my use case, I wouldn't mind if I only had this option programmatically.

### Bug Fix 
The ***ideal workaround*** would be to use sciunit to delete the `e1` kernel in between executing the first notebook with the repeat kernel and executing the second notebook with audit kernel. 
```
sciunit rm e1
```
@sranasir suggested that this should be possible with the most up to date version sciunit. That there was a bug related the execution counter so that even after `e1` is deleted using `sciunit rm e1`, that the next run in the audit kernel produces an `e2` instead of `e1` again, which causes some discrepancy.

###  Current Workaround
Therefore, the ***actual*** current workaround is to manually delete the audit kernel using 
```
rm -rf ~/sciunit/audit-kernel/
./install.sh
```
NOTE: My [install.sh](https://github.com/nicole-brewer/Flinc/blob/chameleon/install.sh) (in the chameleon branch of my personal fork) is slightly modified, but I think the behavior should be the same in both

**Comment by sranasir** [comment]
Thank you for creating the issue as discussed. Meanwhile, I have updated the [README](https://github.com/depaul-dice/Flinc/blob/main/README.md) so only one notebook per Sciunit project is run with audit and repeat kernels.

**Comment by sranasir** [comment]
This issue is resolved using #7 which will be merged in another branch in future.
